projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -4,7 +4,7 @@ import { Auth0ClientService } from './auth.client';
 import { WindowService } from './window';
 import { Auth0Client } from '@auth0/auth0-spa-js';
 import { AbstractNavigator } from './abstract-navigator';
-import { toArray, filter } from 'rxjs/operators';
+import { toArray, filter, concatMap } from 'rxjs/operators';
```

Comment:
`concatMap` is not used in the diff introduced on this PR. Is this import valid?

Reply:
Nope! Removed :)